build/networks.go

```diff
@@ -0,0 +1,51 @@
+package build
+
+import "errors"
+
+// GetNetwork returns the network of a current build.
+func GetNetwork() Network {
+	return network
+}
+
+// DefaultNetwork sets a default network for Celestia Node.
+var DefaultNetwork = DevNet
+
+// NOTE: Everytime we add a new long-running network, it has to be added it here.
+const (
+	// DevNet or devnet-2 according to celestiaorg/networks
+	DevNet Network = "devnet-2"
+)
+
+// Network is a type definition for DA network run by Celestia Node.
+type Network string
+
+// ErrInvalidNetwork is thrown when unknown network is used.
+var ErrInvalidNetwork = errors.New("build: invalid network")
+
+// Validate the network.
+func (n Network) Validate() error {
+	if _, ok := networksList[n]; !ok {
+		return ErrInvalidNetwork
+	}
+	return nil
+}
+
+// A strict list of networks.
+var networksList = map[Network]struct{}{
+	DevNet: {},
+}
+
+// A used network within a build.
+// Can be set with 'ldflags'
+var network Network
```

Comment:
```suggestion
// network is the currently used network within a build.
// It can be set with 'ldflags'.
var network Network
```

Reply:
Idflags?

Reply:
[This](https://github.com/celestiaorg/celestia-node/blob/main/Makefile#L3) and [this](https://github.com/celestiaorg/celestia-node/blob/main/Makefile#L14). The way we pass build-time flags in Go...

Reply:
can you add: 

```
It can be set with `-ldflags` during build.
```

Reply:
ldflags implies during build. Dash ahead does make any difference.